app/script/media/MediaStreamHandler.js

```diff
@@ -268,11 +268,6 @@ z.media.MediaStreamHandler = class MediaStreamHandler {
     return constraintsPromise
       .then(streamConstraints => this.requestMediaStream(mediaType, streamConstraints))
       .then(mediaStreamInfo => {
-        if (!this.mediaStreamInUse()) {
```

Comment:
@gregor I cannot really find a reason for that condition. Maybe you have insights regarding this.

This prevents the preferences page to update the stream shown when switching inputs